napari/components/dims.py

```diff
@@ -1,265 +1,220 @@
+        List of tuples (min, max, step), one for each dimension. In a world
+        coordinates space.
+    current_step : tuple of int
+        Tuple the slider position for each dims slider, in slider coordinates.
+    order : tuple of int
+        Tuple of ordering the dimensions, where the last dimensions are rendered.
+    axis_labels : tuple of str
+        Tuple of labels for each dimension.
 
     Attributes
     ----------
-    events : EmitterGroup
-        Event emitter group
-    range : list of 3-tuple
+    ndim : int
+        Number of dimensions.
+    ndisplay : int
+        Number of displayed dimensions.
+    last_used : int
+        Dimension which was last used.
+    range : tuple of 3-tuple of float
         List of tuples (min, max, step), one for each dimension. In a world
         coordinates space.
-    point : list of float
-        List of floats setting the current value of the range slider when in
-        POINT mode, one for each dimension. In a world coordinates space.
     current_step : tuple of int
         Tuple the slider position for each dims slider, in slider coordinates.
+    order : tuple of int
+        Tuple of ordering the dimensions, where the last dimensions are rendered.
+    axis_labels : tuple of str
+        Tuple of labels for each dimension.
     nsteps : tuple of int
-        Number of steps available to each slider.
-    ndim : int
-        Number of dimensions.
-    displayed : tuple
-        List of dimensions that are displayed.
-    not_displayed : tuple
-        List of dimensions that are not displayed.
-    displayed_order : tuple
-        Order of only displayed dimensions.
+        Number of steps available to each slider. These are calculated from
+        the ``range``.
+    point : tuple of float
+        List of floats setting the current value of the range slider when in
+        POINT mode, one for each dimension. In a world coordinates space. These
+        are calculated from the ``current_step`` and ``range``.
+    displayed : tuple of int
+        List of dimensions that are displayed. These are calculated from the
+        ``order`` and ``ndisplay``.
+    not_displayed : tuple of int
+        List of dimensions that are not displayed. These are calculated from the
+        ``order`` and ``ndisplay``.
+    displayed_order : tuple of int
+        Order of only displayed dimensions. These are calculated from the
+        ``displayed`` dimensions.
     """
 
-    def __init__(self, ndim=None, *, ndisplay=2, order=None, axis_labels=None):
-        super().__init__()
-
-        # Events:
-        self.events = EmitterGroup(
-            source=self,
-            auto_connect=False,
-            current_step=None,
-            axis_labels=None,
-            ndim=None,
-            ndisplay=None,
-            order=None,
-            range=None,
-            last_used=None,
-            deprecated={"axis": "current_step", "camera": "ndisplay"},
```

Comment:
So now there is no deprecation mechanism, so `camera` is dropped?

Reply:
That's correct, those both were deprecated and now are being dropped.